src/assets/i18n/es.json

```diff
@@ -7,6 +7,8 @@
     "spanish": "ES"
   },
   "links": {
-    "about": "Sobre Nosotras"
+    "about": "Sobre Nosotras",
+    "who did this": "QUIEN HIZO ESTO?",
```

Comment:
Let's add accents and also opening question mark is missin

Reply:
Done